dash/backend/src/modules/kubesec/services/kubesec.service.ts

```diff
@@ -36,17 +36,13 @@ export class KubesecService {
         return podList;
     }
 
-    async runKubesecByPod(podName: string, namespace: string, clusterId: number): Promise<any> {
+    async runKubesecByPod(podName: string, namespace: string, clusterId: number): Promise<{ data: KubesecScanResults }> {
         const cluster = await this.clusterService.getClusterById(clusterId);
         if (cluster) {
```

Comment:
TLDR: removing the try-catch is not a regression.

The try-catch here effectively caused this to return the string "Pod couldn't be reached". The only previous usage of this function was the kube sec controller, which wouold then immediately error trying to access a property of that string. So removing the try-catch causes it to catch the real error at that point.